standard/tier/tier_data.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			order = 1,
+			value = '1',
+			sort = 'A1',
```

Comment:
Is `sort` needed? Will there be cases where the sorting on `order` differs from sorting on `sort`?

Reply:
it is the hidden sort value
hidden sort values continue from tier to tiertype